src/functions-and-arrays.js

```diff
@@ -1,73 +1,480 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(numOne, numTwo) {
+  if (numOne > numTwo) {
+    return numOne;
+  } else if (numTwo > numOne) {
+    return numTwo;
+  } else {
```

Comment:
If you use >= in one of the above cases then you would not need to check this last condition. Less code for good :)